fft/src/KokkosFFT.hpp

```diff
@@ -2,8 +2,8 @@
 //
 // SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
 
-#ifndef KOKKOS_FFT_HPP
-#define KOKKOS_FFT_HPP
+#ifndef KOKKOSFFT_HPP
```

Comment:
Maybe better. I can do that with python

Reply:
If it can help, I did that in DDC https://github.com/CExA-project/ddc/blob/dc487b51b3ddd17402c25c05633ff94c8cfc0766/.github/workflows/tests.yml#L25-L26.

We could propose a github action to help Kokkos based projects